third_party/autograph/autograph/impl/api.py

```diff
@@ -142,56 +128,58 @@ def _attach_error_metadata(e, f):
                                        __file__)
 
 
-class StackTraceMapper(tf_stack.StackTraceMapper):
-  """Remaps generated code to code it originated from."""
+# class StackTraceMapper(tf_stack.StackTraceMapper):
```

Comment:
Do you intend to revive this piece?

Reply:
This is the piece mentioned in the PR description about stack remapping. I would need to plug it into DALI build, so I will wait with it till I move it into DALI, integrate the tests etc.